BO_ 171 NEW_MSG_AB: 8 XXX
 SG_ NEW_SIGNAL_5 : 3|12@0+ (1,0) [0|4095] "" XXX
 SG_ NEW_SIGNAL_6 : 21|6@0+ (1,0) [0|63] "" XXX
 SG_ NEW_SIGNAL_1 : 23|2@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_2 : 35|4@0+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 51|12@0+ (1,0) [0|4095] "" XXX
 SG_ NEW_SIGNAL_4 : 55|4@0+ (1,0) [0|15] "" XXX

BO_ 222 EPS_1: 6 EPS
 SG_ STEERING_ANGLE : 5|14@0+ (0.1,-718.7) [0|16383] "deg" XXX
 SG_ NEW_SIGNAL_1 : 6|2@1+ (1,0) [0|3] "" XXX
 SG_ STEERING_RATE : 19|12@0+ (0.5,-1000) [0|4095] "deg/s" XXX
 SG_ COUNTER : 35|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 47|8@0+ (1,0) [0|255] "" XXX

BO_ 228 NEW_MSG_E4: 6 XXX
 SG_ NEW_SIGNAL_1 : 13|1@0+ (1,0) [0|1] "" XXX

BO_ 238 ABS_1: 8 ABS
 SG_ WHEEL_SPEED_FL : 7|13@0+ (0.017,0) [0|8191] "m/s" XXX
 SG_ WHEEL_SPEED_FR : 10|13@0+ (0.017,0) [0|8191] "m/s" XXX
 SG_ WHEEL_SPEED_RL : 29|13@0+ (0.017,0) [0|8191] "m/s" XXX
 SG_ WHEEL_SPEED_RR : 32|13@0+ (0.017,0) [0|8191] "m/s" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 241 NEW_MSG_F1: 8 XXX
 SG_ MAYBE_VOLTAGE : 18|10@0+ (0.02,0) [0|1023] "" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 249 NEW_MSG_F9: 4 XXX
 SG_ COUNTER : 19|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX

BO_ 250 ABS_3: 8 ABS
 SG_ BRAKE_PRESSURE_THRESHOLD : 2|1@0+ (1,0) [0|1] "" XXX
 SG_ BRAKE_PEDAL_SWITCH : 3|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 4|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 9|2@0+ (1,0) [0|3] "" XXX
 SG_ XCOUNTER : 38|4@0+ (1,0) [0|15] "" XXX
 SG_ XCHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 251 NEW_MSG_FB: 8 XXX
 SG_ NEW_SIGNAL_3 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_6 : 16|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_4 : 23|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_5 : 28|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_7 : 32|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_8 : 40|3@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_1 : 48|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_9 : 53|3@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_2 : 56|8@1+ (1,0) [0|255] "" XXX

BO_ 252 ENGINE_1: 8 ENGINE
 SG_ ENGINE_RPM : 7|14@0+ (1,0) [0|255] "rev/min" XXX
 SG_ ACCEL_PEDAL : 20|8@0+ (0.4,0) [0|255] "percent" XXX
 SG_ NEW_SIGNAL_1 : 22|2@0+ (1,0) [0|3] "" XXX
 SG_ REVERSE : 26|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 40|7@1+ (1,0) [0|127] "" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_4 : 52|4@1+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 254 ABS_2: 8 ABS
 SG_ LONG_ACCEL : 7|12@0+ (0.01,-20.48) [0|4095] "m/s2" XXX
 SG_ LATERAL_ACCEL : 11|12@0+ (0.01,-20.48) [0|4095] "m/s2" XXX
 SG_ YAW_RATE : 31|12@0+ (-0.0014,2.86) [0|4095] "rad/s" XXX
 SG_ NEW_SIGNAL_1 : 47|9@0+ (1,0) [0|511] "" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 256 NEW_MSG_100: 8 XXX
 SG_ NEW_SIGNAL_1 : 16|3@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_2 : 19|2@1+ (1,0) [0|3] "" XXX
 SG_ COUNTER : 48|4@1+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 56|8@1+ (1,0) [0|255] "" XXX

BO_ 257 ABS_6: 8 ABS
 SG_ MAYBE_ACC_BRAKE : 5|1@0+ (1,0) [0|1] "" XXX
 SG_ MAYBE_VEHICLE_SPEED : 15|11@0+ (0.0602,0) [0|2047] "km/h" XXX
 SG_ BRAKE_PRESSURE_1 : 20|11@0+ (1,0) [0|2047] "" XXX
 SG_ NEW_SIGNAL_1 : 43|1@0+ (1,0) [0|1] "" XXX
 SG_ COUNTER : 48|4@1+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 262 EPS_2: 7 EPS
 SG_ UNKNOWN_TORQUE : 7|12@0+ (1,-2000) [0|4095] "" XXX
 SG_ UNKNOWN_1 : 8|1@0+ (1,0) [0|1] "" XXX
 SG_ UNKNOWN_STATUS : 9|1@0+ (1,0) [0|1] "" XXX
 SG_ DRIVER_TORQUE : 23|11@0+ (1,-1024) [0|2047] "" XXX
 SG_ UNKNOWN_2 : 27|1@0+ (1,0) [0|1] "" XXX
 SG_ LKA_STATUS : 38|2@0+ (1,0) [0|3] "" XXX
 SG_ LKA_FAULT : 39|1@0+ (1,0) [0|1] "" XXX
 SG_ COUNTER : 43|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 55|8@0+ (1,0) [0|255] "" XXX

BO_ 263 ABS_4: 8 ABS
 SG_ BRAKE_PRESSURE : 7|8@0+ (1,0) [0|255] "" XXX

BO_ 265 NEW_MSG_109: 8 XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 270 ABS_7: 7 XXX
 SG_ LONG_ACCEL_RAW : 7|12@0+ (0.01,-20.48) [0|4095] "" XXX
 SG_ LATERAL_ACCEL_RAW : 11|12@0+ (0.01,-20.48) [0|4095] "" XXX
 SG_ YAW_RATE_RAW : 31|12@0+ (-0.0014,2.86) [0|4095] "" XXX
 SG_ COUNTER : 43|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 55|8@0+ (1,0) [0|255] "" XXX

BO_ 278 ABS_5: 8 ABS
 SG_ WHEEL_IMPULSE_FL : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ WHEEL_IMPULSE_FR : 15|8@0+ (1,0) [0|255] "" XXX
 SG_ WHEEL_IMPULSE_RL : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ WHEEL_IMPULSE_RR : 31|8@0+ (1,0) [0|255] "" XXX
 SG_ ACTIVE_FL : 32|1@0+ (1,0) [0|1] "" XXX
 SG_ ACTIVE_FR : 33|1@0+ (1,0) [0|1] "" XXX
 SG_ ACTIVE_RL : 34|1@0+ (1,0) [0|1] "" XXX
 SG_ ACTIVE_RR : 35|1@0+ (1,0) [0|1] "" XXX
 SG_ FORWARD_1 : 36|1@0+ (1,0) [0|1] "" XXX
 SG_ REVERSE_1 : 37|1@0+ (1,0) [0|1] "" XXX
 SG_ FORWARD_2 : 38|1@0+ (1,0) [0|1] "" XXX
 SG_ REVERSE_2 : 39|1@0+ (1,0) [0|1] "" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 280 NEW_MSG_118: 6 XXX
 SG_ COUNTER : 35|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 47|8@0+ (1,0) [0|255] "" XXX

BO_ 282 NEW_MSG_11A: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|11@0+ (1,-1000) [0|2047] "" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 284 NEW_MSG_11C: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|12@0+ (1,0) [0|4095] "" XXX
 SG_ NEW_SIGNAL_2 : 11|12@0+ (1,0) [0|4095] "" XXX
 SG_ NEW_SIGNAL_3 : 28|13@0+ (1,0) [0|8191] "" XXX
 SG_ NEW_SIGNAL_4 : 31|2@0+ (1,0) [0|3] "" XXX
 SG_ VEHICLE_SPEED : 47|12@0+ (0.017,0) [0|4095] "m/s" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 288 NEW_MSG_120: 6 XXX
 SG_ COUNTER : 19|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX

BO_ 290 EPS_3: 4 EPS
 SG_ EPS_TORQUE : 7|12@0+ (1,-2048) [0|4095] "" XXX
 SG_ COUNTER : 19|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX

BO_ 346 NEW_MSG_15A: 4 XXX
 SG_ MAYBE_COUNTER : 16|4@1+ (1,0) [0|15] "" XXX
 SG_ MAYBE_CHECKSUM : 24|8@1+ (1,0) [0|255] "" XXX

BO_ 400 NEW_MSG_190: 8 XXX
 SG_ NEW_SIGNAL_1 : 12|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 13|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_3 : 21|1@0+ (1,0) [0|1] "" XXX

BO_ 496 NEW_MSG_1F0: 8 XXX
 SG_ NEW_SIGNAL_1 : 3|7@0+ (1,0) [0|63] "" XXX

BO_ 501 NEW_MSG_1F5: 5 XXX
 SG_ NEW_SIGNAL_4 : 5|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 22|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_3 : 27|4@0+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 39|8@0+ (1,0) [0|255] "" XXX

BO_ 502 LKAS_COMMAND: 8 CAMERA
 SG_ STEERING_TORQUE : 7|11@0+ (1,-1024) [0|2047] "" XXX
 SG_ LKA_WATCH_STATUS : 12|1@0+ (1,0) [0|1] "" XXX
 SG_ COUNTER : 19|4@0+ (1,0) [0|1] "" XXX
 SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX

BO_ 506 NEW_MSG_1FA: 8 XXX
 SG_ NEW_SIGNAL_1 : 20|4@1+ (1,0) [0|15] "" XXX

BO_ 508 NEW_MSG_1FC: 8 XXX
 SG_ NEW_SIGNAL_1 : 6|1@0+ (1,0) [0|1] "" XXX

BO_ 552 NEW_MSG_228: 8 XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ MAYBE_CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 592 NEW_MSG_250: 4 XXX
 SG_ NEW_SIGNAL_1 : 5|3@1+ (1,0) [0|7] "" XXX

BO_ 599 NEW_MSG_257: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 8|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_3 : 13|3@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_4 : 16|5@1+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_5 : 22|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_6 : 29|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_7 : 31|1@0+ (1,0) [0|1] "" XXX

BO_ 601 NEW_MSG_259: 8 XXX
 SG_ NEW_SIGNAL_2 : 24|5@0+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_1 : 47|8@0+ (1,0) [0|255] "" XXX

BO_ 751 NEW_MSG_2EF: 8 XXX
 SG_ NEW_SIGNAL_1 : 8|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 9|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_3 : 10|1@0+ (1,0) [0|1] "" XXX

BO_ 762 NEW_MSG_2FA: 3 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 16|8@1+ (1,0) [0|255] "" XXX

BO_ 766 CAM_UNKNOWN_1: 4 CAMERA

BO_ 900 BUTTONS: 8 XXX
 SG_ MAYBE_START : 1|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_1 : 19|3@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_2 : 27|1@0+ (1,0) [0|1] "" XXX
 SG_ LKAS_BUTTON : 30|1@1+ (1,0) [0|3] "" XXX
 SG_ POISON : 53|1@0+ (1,0) [0|1] "" XXX

BO_ 999 NEW_MSG_3E7: 6 XXX
 SG_ NEW_SIGNAL_8 : 3|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_5 : 6|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_3 : 8|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_4 : 10|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_7 : 13|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_6 : 20|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 36|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 45|2@1+ (1,0) [0|3] "" XXX

BO_ 1040 NEW_MSG_410: 8 XXX
 SG_ NEW_SIGNAL_3 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_1 : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 49|1@0+ (1,0) [0|1] "" XXX
 SG_ COUNTER : 55|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 1042 NEW_MSG_412: 5 XXX

BO_ 1046 NEW_MSG_416: 8 XXX
 SG_ MUX M : 1|2@0+ (1,0) [0|3] "" XXX
 SG_ UNKNOWN_M0_1 m0 : 15|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M0_2 m0 : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M0_3 m0 : 31|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M0_4 m0 : 39|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M0_5 m0 : 47|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M0_6 m0 : 55|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M0_7 m0 : 63|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_1 m1 : 15|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_2 m1 : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_3 m1 : 31|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_4 m1 : 39|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_5 m1 : 47|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_6 m1 : 55|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M1_7 m1 : 63|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_1 m2 : 15|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_2 m2 : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_3 m2 : 31|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_4 m2 : 39|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_5 m2 : 47|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_6 m2 : 55|8@0+ (1,0) [0|255] "" XXX
 SG_ UNKNOWN_M2_7 m2 : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 1047 NEW_MSG_417: 8 XXX
 SG_ NEW_SIGNAL_1 : 29|1@0+ (1,0) [0|1] "" XXX

BO_ 1048 NEW_MSG_418: 4 XXX
 SG_ COUNTER : 19|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX

BO_ 1053 NEW_MSG_41D: 8 XXX
 SG_ NEW_SIGNAL_1 : 16|7@1+ (1,0) [0|127] "" XXX

BO_ 1062 NEW_MSG_426: 8 XXX

BO_ 1194 CAM_UNKNOWN_2: 8 CAMERA

BO_ 1198 LKA_HUD_1: 8 CAMERA
 SG_ NEW_SIGNAL_1 : 4|5@0+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_3 : 12|5@0+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_2 : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_5 : 29|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_4 : 38|3@0+ (1,0) [0|7] "" XXX
 SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
 SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX

BO_ 1199 NEW_MSG_4AF: 8 XXX
 SG_ NEW_SIGNAL_1 : 2|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 15|1@0+ (1,0) [0|1] "" XXX

BO_ 1202 NEW_MSG_4B2: 8 XXX
 SG_ NEW_SIGNAL_1 : 16|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 30|2@1+ (1,0) [0|3] "" XXX

BO_ 1349 NEW_MSG_545: 8 XXX
 SG_ NEW_SIGNAL_5 : 7|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_6 : 16|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_7 : 18|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_4 : 22|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 29|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_3 : 39|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_1 : 40|3@0+ (1,0) [0|7] "" XXX

BO_ 1351 LKA_HUD_2: 8 CAMERA
 SG_ MAYBE_LKAS_DISABLED : 30|1@0+ (1,0) [0|1] "" XXX
 SG_ LANE_HUD_INDICATOR : 51|3@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_2 : 55|4@0+ (1,0) [0|15] "" XXX

BO_ 1354 NEW_MSG_54A: 8 XXX
 SG_ NEW_SIGNAL_1 : 8|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_2 : 20|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 30|1@0+ (1,0) [0|1] "" XXX

BO_ 1445 CRUISE_BUTTONS: 8 XXX
 SG_ ACC_ACTIVE : 7|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_1 : 19|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 20|1@0+ (1,0) [0|1] "" XXX

BO_ 1446 NEW_MSG_5A6: 8 XXX
 SG_ NEW_SIGNAL_1 : 24|5@1+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_2 : 46|1@0+ (1,0) [0|1] "" XXX

BO_ 1448 GEAR: 8 XXX
 SG_ PARKING_BREAK : 4|1@0+ (1,0) [0|1] "" XXX
 SG_ MAYBE_P_OR_R : 7|1@0+ (1,0) [0|1] "" XXX
 SG_ MAYBE_D : 16|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_1 : 21|1@0+ (1,0) [0|7] "" XXX
 SG_ REVERSE : 22|1@0+ (1,0) [0|1] "" XXX
 SG_ MAYBE_N : 25|2@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_3 : 30|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_4 : 31|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_6 : 32|3@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_7 : 39|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 47|2@0+ (1,0) [0|1] "" XXX
 SG_ MAYBE_COUNTER : 48|4@1+ (1,0) [0|15] "" XXX
 SG_ MAYBE_CHECKSUM : 56|8@1+ (1,0) [0|255] "" XXX

BO_ 1449 NEW_MSG_5A9: 8 XXX
 SG_ NEW_SIGNAL_2 : 12|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_1 : 21|3@1+ (1,0) [0|7] "" XXX

BO_ 1456 NEW_MSG_5B0: 8 XXX
 SG_ MAYBE_PARKING_BREAK : 13|1@0+ (1,0) [0|1] "" XXX

BO_ 1458 CAM_UNKNOWN_5: 4 CAMERA
 SG_ NEW_SIGNAL_1 : 5|1@0+ (1,0) [0|1] "" XXX

BO_ 1504 NEW_MSG_5E0: 8 XXX
 SG_ NEW_SIGNAL_1 : 19|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_2 : 22|1@0+ (1,0) [0|1] "" XXX

BO_ 1538 NEW_MSG_602: 8 XXX
 SG_ MAYBE_POISON : 7|1@0+ (1,0) [0|1] "" XXX

BO_ 1846 NEW_MSG_736: 8 XXX
 SG_ NEW_SIGNAL_1 : 2|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_2 : 17|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_3 : 19|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_4 : 24|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_5 : 32|4@1+ (1,0) [0|15] "" XXX

BO_ 1848 NEW_MSG_738: 5 XXX
 SG_ NEW_SIGNAL_1 : 9|1@0+ (1,0) [0|1] "" XXX

BO_ 1850 NEW_MSG_73A: 6 XXX
 SG_ NEW_SIGNAL_1 : 8|1@0+ (1,0) [0|1] "" XXX

BO_ 1854 BCM_1: 4 BCM
 SG_ LEFT_SIGNAL : 15|1@0+ (1,0) [0|1] "" XXX
 SG_ RIGHT_TURN_STALK : 16|1@0+ (1,0) [0|1] "" XXX
 SG_ LEFT_TURN_STALK : 17|1@0+ (1,0) [0|1] "" XXX
 SG_ RIGHT_SIGNAL : 21|1@0+ (1,0) [0|1] "" XXX

BO_ 1865 NEW_MSG_749: 8 XXX
 SG_ NEW_SIGNAL_1 : 23|2@0+ (1,0) [0|3] "" XXX

BO_ 417001768 NEW_MSG_18DAF128: 8 XXX

BO_ 417001776 NEW_MSG_18DAF130: 4 XXX

BO_ 417002024 NEW_MSG_18DAF228: 8 XXX

BO_ 506724352 NEW_MSG_1E340000: 8 XXX
 SG_ NEW_SIGNAL_2 : 9|1@0+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_1 : 40|1@0+ (1,0) [0|1] "" XXX

BO_ 506855454 CAM_UNKNOWN_6: 1 CAMERA

CM_ SG_ 254 LONG_ACCEL "scale TBD";
CM_ SG_ 254 LATERAL_ACCEL "scale TBD";
CM_ SG_ 254 YAW_RATE "scale estimated";
CM_ SG_ 257 MAYBE_ACC_BRAKE "may correlate with ACC-actuated braking";
CM_ SG_ 278 FORWARD_1 "probably per-axle";
CM_ SG_ 278 REVERSE_1 "probably per-axle";
CM_ SG_ 278 FORWARD_2 "probably per-axle";
CM_ SG_ 278 REVERSE_2 "probably per-axle";
CM_ SG_ 282 NEW_SIGNAL_1 "smoothed yaw rate with low speed cutoff, maybe active forward lighting target angle";
CM_ SG_ 284 VEHICLE_SPEED "scale estimated";
CM_ SG_ 502 COUNTER "correlates with steering wheel haptic";
CM_ BO_ 1198 "Definite LKA activity, probably contains lane-line recognition and lane departure signals, indicator of LKA vs TJA, perhaps indicator of active assist in map-permitted areas";
VAL_ 262 LKA_STATUS 0 "standby" 1 "lka_active";
VAL_ 502 LKA_WATCH_STATUS 0 "OFF" 1 "ON";
VAL_ 1351 LANE_HUD_INDICATOR 0 "DISABLED" 1 "ENABLED" 2 "UNKNOW_1";
